lbrynet/core/utils.py

```diff
@@ -9,6 +9,8 @@
 
 import pkg_resources
 
+from twisted.internet import reactor
```

Comment:
reactortest1.py
http://pastebin.com/6HPWQTBb
reactortest2.py
http://pastebin.com/gcvKnFUe

Doesn't look like a problem.  reactortest2.py is fully aware of the added system trigger added in reactortest1.py